chingu-coders/Voyage2-Turtles-02
# persist gallery bg choice, edit sticky note title

## Purpose
* Improve user features

## Approach/Fixes
* Used localStorage to store gallery bg image chosen
* Added 'cancel' button for sticky notes

## Testing
* Pull code to local machine
* Choose an image from the gallery and refresh the page the see that it persists
* Choose an image from the random generator or upload to see that the background from the gallery no longer persists after reload
* Check that the sticky note top bar has different styling (semi-transparent title and icons, white on hover)
* Change the title of a sticky note and test the cancel
* If note title has never been changed, it should default to "Sticky Note" otherwise it should default to the last changed title
